stestr/tests/test_return_codes.py

```diff
@@ -199,3 +201,16 @@ def test_load_from_stdin(self):
         stream = self._get_cmd_stdout(
             'stestr last --subunit')[0]
         self.assertRunExit('stestr load', 0, stdin=stream)
+
+    def test_load_from_stdin_quiet(self):
+        out, err = self.assertRunExit('stestr -q run passing', 0)
+        self.assertEqual(out.decode('utf-8'), '')
+        # FIXME(masayukig): We get some warnings when we run a coverage job.
+        # So, just ignore 'err' here.
+        stream = self._get_cmd_stdout(
+            'stestr last --subunit')[0]
+        out, err = self.assertRunExit('stestr -q load', 0, stdin=stream)
+        # FIXME(masayukig): I expected the 'out' was empty, however, we get
+        # some standard outputs, actually such as
+        # 'Ran 79 tests in 5.930s\nPASSED (id=150)'.
```

Comment:
I feel like this is a bug, we should be passing the stdout variable to all the output helper functions. If something is leaking to stdout then we're probably missing that somewhere.

Reply:
heh, yeah, I'll dig it. A unit test is important :)